Libraries/Opc.Ua.Client/SessionClientBatched.cs

```diff
@@ -59,7 +59,7 @@ public SessionClientBatched(ITransportChannel channel)
         public OperationLimits OperationLimits
         {
             get => m_operationLimits;
-            internal set
+            protected set
```

Comment:
maybe it has to be public to fix the tests ...

Reply:
would internal protected work as well?

Reply:
Unless this causes any issues I would second this change. (Internal protected as well...)

Reply:
updated to protected internal